doc/source/tutorial/examples/gaussian_filter_plot1.py

```diff
@@ -0,0 +1,62 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import matplotlib as mpl
+from scipy.ndimage import gaussian_filter
+
+grids = 2
+boxs = 5
+x, y, z = np.indices((boxs * grids, boxs * grids, boxs * grids))
+
+def cube(pos=(0, 0, 0)):
+    o1 = pos[0]
+    o2 = pos[1]
+    o3 = pos[2]
+    cube = (
+        (o1 <= x)
+        & (x < o1 + boxs)
+        & (o2 <= y)
+        & (y < o2 + boxs)
+        & (o3 <= z)
+        & (z < o3 + boxs)
+    )
+    return cube
+
+
+voxelarray = np.zeros((boxs * grids, boxs * grids, boxs * grids))
+
+voxelarray[cube((0, 0, 0))] = 1 * 255 / 8
+voxelarray[cube((boxs, 0, 0))] = 2 * 255 / 8
+voxelarray[cube((0, boxs, 0))] = 3 * 255 / 8
+voxelarray[cube((boxs, boxs, 0))] = 4 * 255 / 8
+voxelarray[cube((0, 0, boxs))] = 5 * 255 / 8
+voxelarray[cube((boxs, 0, boxs))] = 6 * 255 / 8
+voxelarray[cube((0, boxs, boxs))] = 7 * 255 / 8
+voxelarray[cube((boxs, boxs, boxs))] = 8 * 255 / 8
```

Comment:
I guess the idea is to convey an instructional/educational pattern in the source here instead of using the simplest/"fastest" expression like `255` instead of `8 * 255 / 8` or `255 / 2` instead of `4 * 255 / 8`?

Or maybe it just doesn't matter much. I think Pamphile was already pretty happy with the final plot in any case, if I remember correctly. Maybe we'll just need to take a peek at the final rendered tutorial (the CI should link to an artifact/web page render when passing).